tests/app/test_route_authentication.py

```diff
@@ -34,15 +34,19 @@ def test_all_routes_have_authentication(client):
         '/auth/logout',
         '/ga4/open-email-tracking/<notification_id>',
         '/internal/<generic>',
+        '/internal/sleep',
+        '/internal/502',
```

Comment:
do we really want to keep these?

Reply:
https://github.com/department-of-veterans-affairs/notification-api/pull/2266#discussion_r1945359958